src/lib.rs

```diff
@@ -0,0 +1,201 @@
+use anyhow::{anyhow, bail, Ok, Result};
+use bb8_redis::{bb8::Pool, redis::cmd, RedisConnectionManager};
+use lazy_static::lazy_static;
+use std::{
+    fs::File,
+    net::IpAddr,
+    path::Path,
+    sync::{Arc, RwLock},
+};
+use tracing::metadata::LevelFilter;
+use tracing_appender::non_blocking::WorkerGuard;
+use tracing_subscriber::{
+    fmt, prelude::__tracing_subscriber_SubscriberExt, util::SubscriberInitExt, EnvFilter, Layer,
+};
+
+#[cfg(test)]
+mod test;
+
+#[macro_export]
```

Comment:
For these 5 macros, I think adding simple examples as a rustdoc would be helpful for future users. Similar documenting style would be found be tracing library, too. Though, I think we can make the rustdoc for these macros very short (like around 5 lines).

Reply:
For the rust doc documentation are you referring something like this : https://doc.rust-lang.org/rustdoc/what-is-rustdoc.html. Or should I just modify the README.

Reply:
Yes the link is what I meant.
Refer to tracing here : https://github.com/tokio-rs/tracing/blob/master/tracing/src/macros.rs#L1264-L1301

Reply:
I wrote a short description and few examples but let me know if I should reword or elaborate more since I wrote it very short and concise

Reply:
Great job! A minor change would be helpful though. 
- /// trace!(LogLocation::Local, "error");   -> /// trace!(LogLocation::Local, "trace");
- /// debug!(LogLocation::Local, "error");  -> /// debug!(LogLocation::Local, "debug");
- an so on

Reply:
Done, I did not catch that when I was writing the docs